db/events.php

```diff
@@ -46,17 +46,5 @@
                 'eventname' => '\core\event\capability_unassigned',
                 'callback' => 'mod_studentquiz_observer::capability_unassigned',
         ],
-        [
-                'eventname' => '\core\event\role_assigned',
-                'callback' => 'mod_studentquiz_observer::role_assigned',
-        ],
-        [
-                'eventname' => '\core\event\role_unassigned',
-                'callback' => 'mod_studentquiz_observer::role_unassigned',
-        ],
```

Comment:
we can't get rid of those, see my comment https://github.com/frankkoch/moodle-mod_studentquiz/pull/309#issuecomment-782301910